scipy/stats/_stats_py.py

```diff
@@ -9042,14 +9051,58 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     :math:`\mathbb{R} \times \mathbb{R}` whose marginals are :math:`u` and
     :math:`v` on the first and second factors respectively.
 
-    If :math:`U` and :math:`V` are the respective CDFs of :math:`u` and
-    :math:`v`, this distance also equals to:
+    In the 1-dimensional case, let :math:`U` and :math:`V` denote the
+    respective CDFs of :math:`u` and :math:`v`, this distance also equals to:
 
     .. math::
 
         l_1(u, v) = \int_{-\infty}^{+\infty} |U-V|
 
-    See [2]_ for a proof of the equivalence of both definitions.
+    See [3]_ for a proof of the equivalence of both definitions.
+
+    In the more general(higher dimensional) and discrete case, it is also
+    called the optimal transport problem or the Monge problem, which can be
+    expressed in this form,
+
+    Given :math:`\Gamma` the transport plan, :math:`D` the distance matrix and
+    follows,
+
+    .. math:
```

Comment:
```suggestion
    .. math::
```
I think that's the problem.

Reply:
Thank you! I will try to fix all and come back to you next week.